docs/.vuepress/config/locales/it/nav.js

```diff
@@ -0,0 +1,13 @@
+module.exports = [
+    { text: 'Guide', link: '/' },
+    { text: 'Extend', link: '/extend/' },
```

Comment:
you can keep extend docs link `/extent` before you translated extend docs, this will link readers to the English document